contrib/mandoc/config.h

```diff
@@ -13,7 +13,7 @@
 #define HAVE_ENDIAN 0
 #define HAVE_ERR 1
 #define HAVE_FTS 1
-#ifdef __GLIBC__
+#ifdef __linux__
```

Comment:
This likely should be upstraemed, but it changes a local change, so it's OK.Both the changes need to go up.... :)